### ISSUE TITLE
Issue with JSON Serializable output

### ISSUE THREAD

**Issue by johnnyfekete** [MarcinOrlowski/laravel-api-response-builder#141]
I'm not sure this error only occurs to me, or in general, but the JsonSerializable converter doesn't work.

I think the problem is here in line 35:

```
return ['val' => json_decode($obj->jsonSerialize(), true)];
```

Where json_decode is sent on an object (as far as I understand, json_decode would create a JSON from a string, while $obj->jsonSerialize() cleans the object to make it ready for converting it to JSON).

changing the line to
```
return ['val' => json_decode(json_encode($obj->jsonSerialize()), true)];
```
solves the issue

**Comment by MarcinOrlowski** [comment]
adding `json_encode()` fixes only one specific case you are facing (but will break unit tests). Can you please provide more information about what is the data that is `JsonSerializable` that triggers this error?

**Comment by MarcinOrlowski** [comment]
Fixed in v7.1.2.
